stock_prices/stock_prices.py

```diff
@@ -2,14 +2,40 @@
 
 import argparse
 
+
 def find_max_profit(prices):
```

Comment:
ran this code and it's working fine!
you can still trim this down quite a bit. when you're done with MVP, come back to this and see if you can get rid of a few `if-else` clauses